sdk/python/feast/on_demand_feature_view.py

```diff
@@ -47,6 +48,8 @@ class OnDemandFeatureView(BaseFeatureView):
     input_feature_view_projections: Dict[str, FeatureViewProjection]
     input_request_data_sources: Dict[str, RequestDataSource]
     udf: MethodType
+    created_timestamp: Optional[datetime] = None
```

Comment:
The args above arent updated btw

Reply:
What do you mean? The other FeatureView classes don't take it in `__init__`. I was just making sure they all matched in terms of behaviour.

Arguably, this setting of `created_timestamp` to `None` should maybe be inherited behaviour.

Reply:
I mean the docstring. I also think this logic should be inherited.

Reply:
Still not sure what you mean 🤔 The Docstring shouldn't mention this should it as it isn't an arg?

Reply:
Turns out this logic was already performed by the parent class. The logic must've been left in accidentally.